tests/regression/rhbz-1584461/task.yaml

```diff
@@ -19,7 +19,7 @@ restore: |
     rm -f ./in-snap ./on-host
 
 execute: |
-    # on Fedora, /etc/nsswitch.conf is a symlink to /etc/authselect/nsswitch.conf
+    # on Fedora 30 and 31, /etc/nsswitch.conf is a symlink to /etc/authselect/nsswitch.conf
```

Comment:
How about F32?

Reply:
I've dropped any checks for nsswitch, the test even didn't use that, but rather set up a separate symlink of /etc/ssl